ns3_adapter/src/ns3_adapter.cpp

```diff
@@ -248,7 +260,7 @@ void NS3Adapter::pre_spin()
     // Adjust output queue size if config changed.
     if(ns3_client_error_)
     {
-        ROS_ERROR_STREAM("NS-3 Client Error : " << ns3_client_error_.message());
+        ROS_WARN_STREAM("NS-3 Client Error : " << ns3_client_error_.message());
```

Comment:
Closing client did not seem to stop the node from spinning, so the client would start again. So I thought it would be slight misleading if ERROR showed. I included more logs to say it is reconnecting